crypto/src/main/java/net/corda/v5/crypto/SecureHash.java

```diff
@@ -38,5 +38,5 @@ public interface SecureHash {
      * Example outcome of toString(): SHA-256:98AF8725385586B41FEFF205B4E05A000823F78B5F8F5C02439CE8F67A781D90
      */
     @NotNull
-    String toString();
+    String getAlgorithmAndHexString();
```

Comment:
This and its inverse parse are our standard naming/logging convention so don't change this.